pyproject.toml

```diff
@@ -1,8 +1,36 @@
+[tool.poetry]
+name = "pylambertw"
+version = "0.0.4"
+description = "Python implementation of the Lambert W x F framework for analyzing skewed, heavy-tailed distribution with an sklearn interface and torch based maximum likelihood estimation (MLE)."
+authors = ["Georg M. Goerg <im@gmge.org>"]
+homepage = "https://github.com/gmgeorg/pylambertw.git"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+
+numpy = ">=1.0.1"
```

Comment:
see if you require the cython requirement (new requirements.txt file) .  w/o it I had trouble with scipy CI/CD tests passing; if it passes w/o cython requirement as-is, feel free to drop that requirement

Reply:
Ok I tried it with and without that requirement and it seems the CI/CD pipeline passes in both cases. I'll leave it out for now, and if it becomes a problem later you can always add it in.